docs/src/index.md

```diff
@@ -0,0 +1,15 @@
+# DataInterpolations.jl
+
+This library is meant for interpolating one dimensional data by methods which
+both hit every point exactly or smooth the data into an approximate fitting
+curve. The library contains the following interpolation methods:
+
+-  Linear Interpolation
+-  Quadratic Interpolation
+-  Lagrange Interpolation
+-  Zero Spline
+-  Quadratic Spline
+-  Cubic Spline
+-  BSpline Global Curve Interpolation
+-  BSpline Global Curve Approximation
+-  Curve fitting
```

Comment:
Use the README? There's a lot more information that you can just copy-paste.

Reply:
Yes, done in [c75828a](https://github.com/PumasAI/DataInterpolations.jl/pull/173/commits/c75828a02ffe6c1d6be75f900dcb190031bf33d8)